config/members-crossplane.yaml

```diff
@@ -0,0 +1,263 @@
+apiVersion: organizations.github.crossplane.io/v1alpha1
+kind: Membership
+metadata:
+  name: bassam-crossplane
+spec:
+  forProvider:
+    inviteeId: 1463491
+    user: bassam
+    organization: crossplane
+    role: admin
+---
+apiVersion: organizations.github.crossplane.io/v1alpha1
+kind: Membership
+metadata:
+  name: Chrisupbound-crossplane
```

Comment:
I think these names will need to be lowercase.

Reply:
Good catch :)